diff --git a/kernel/include/ksu.h b/kernel/include/ksu.h
index 4af4acb..d8bd9c2 100755
--- a/kernel/include/ksu.h
+++ b/kernel/include/ksu.h
@@ -13,12 +13,8 @@ extern bool allow_shell;
 extern struct selinux_policy *backup_sepolicy;
 extern bool ksu_no_custom_rc;
 
-#ifdef CONFIG_ANDROID
 #include <linux/security.h>
 #define ksu_security_secctx_to_secid security_secctx_to_secid
-#else
-int ksu_security_secctx_to_secid(const char *secdata, u32 seclen, u32 *secid);
-#endif
 
 static inline int startswith(char *s, char *prefix)
 {
