diff --git a/kernel/Kbuild b/kernel/Kbuild
index 96e164fb..3b933eef 100644
--- a/kernel/Kbuild
+++ b/kernel/Kbuild
@@ -11,6 +11,7 @@ kernelsu-objs += infra/file_wrapper.o
 kernelsu-objs += infra/event_queue.o
 kernelsu-objs += infra/seccomp_cache.o
 kernelsu-objs += infra/su_mount_ns.o
+kernelsu-objs += infra/symbol_resolver.o
 
 ifneq ($(CONFIG_KSU_DISABLE_MANAGER),y)
 kernelsu-objs += manager/apk_sign.o
